pynicotine/userbrowse.py

```diff
@@ -119,11 +110,7 @@ def browse_user(self, username, path=None, local_shares_type=None, new_request=F
             user_share.clear()
 
         if username == (config.sections["server"]["login"] or "Default"):
-            if local_shares_type == "normal":
-                self.browse_local_public_shares(path, new_request)
-                return
-
-            self.browse_local_buddy_shares(path, new_request)
+            self.browse_local_shares(path, local_shares_type or "buddy", new_request)
```

Comment:
```suggestion
            self.browse_local_shares(path, local_shares_type, new_request)
```

Reply:
and local_shares_type="buddy" instead of None

Reply:
Assign a default value "buddy" to the local_shares_type parameter instead of None.